tests/test_pipeline.py

```diff
@@ -104,6 +79,8 @@ def test_pipeline_name_in_the_tab(self):
         assert (self.pipelineName_valid in driver.get_element_text(tab_locator_valid))
         assert driver.is_element_present(tab_locator_valid)
 
+
```

Comment:
Are you sure that additional lines are necessary?

Reply:
Ok, I will remove second assert as not necessarily.